compiler/test/compilable/test24280.i

```diff
@@ -4,3 +4,17 @@ struct timespec
 {
     int s;
 };
+
+/* https://issues.dlang.org/show_bug.cgi?id=24303 */
+
+typedef struct {} Slice;
+
+struct Lang
+{
+    Slice *slices;
+};
+
+void langmap(struct Lang *self)
+{
+    Slice slice = *self->slices;
+}
```

Comment:
This part wasn't in the bugzilla test case, but I assume it still reproduces the issue.